css/style.css

```diff
@@ -0,0 +1,555 @@
+  }
+
+  .container__button {
+    font-size: 20px;
+  }
+
+  .title {
+    font-size: 30px;
+  }
+
+  .sneakers_dekstop {
+    display: none;
+  }
+
+  .products__items_border {
+    display: none;
+  }
+
+  .sneakers__image {
+    width: 230px;
+    height: 233px;
+    padding-bottom: 30px;
+  }
+
+  .sneakers__name {
+    font-size: 20px;
+  }
+
+  .sneakers__description {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex: 1;
+  }
+ 
+  .sneakers__description_price {
+    font-size: 25px;
+    flex: 4;
+  }
+
+  .sneakers__description_colors {
+    flex: 2;
+  }
+
+  .color {
+    display: none;
+  }
+
+  .color_mobile {
+    width: 20px;
+    height: 20px;
+    display: flex;
+  }
+
+  .history {
+    flex-direction: column;
+    height: 590px;
+  }
+
+  .history__video {
+    order: 1;
+  }
+
+  .history__text {
+    width: 683px;
+    text-align: center;
+  }
+
+  .history__text_main {
+    font-size: 20px;
+    font-weight: normal;
+    padding-bottom: 30px;
+  }
+
+  .history__button {
+    display: none;
+  }
+
+  .history__button_arrow {
+    display: none;
+  }
+
+  .history__video_main {
+    width: 681px;
+    height: 374px;
+    padding: 0;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  body {
+    max-width: 375px;
+    margin: 0 auto;
+  }
+
+  .header__container {
+    background-size: 706px 191px;
+    height: 191px;
+    padding-left: 50px;
+  }
+
+  .header__logo_mobile {
+    width: 104px;
+    height: 22px;
+    padding-right: 0;
+  }
+
+  .menu__burger {
+    width: 41px;
+    height: 15px;
+  }
+
+  .header__menu {
+    height: 72px;
+  }
+
+  .container__title {
+    font-size: 20px;
+  }
+
+  .container__button {
+    font-size: 12px;
+  }
+
+  .title {
+    font-size: 20px;
+  }
+
+  .products__title {
+    padding: 15px 0 0 0;
+  }
+
+  .sneakers_tablet {
+    display: none;
+  }
+
+  .sneakers__image {
+    width: 166px;
+    height: 163px;
+    align-self: center;
+    padding-bottom: 0;
+  }
+
+  .sneakers__name {
+    font-size: 15px;
+  }
+
+  .sneakers__description {
+    padding-bottom: 50px;
+  }
+
+  .sneakers__description_price {
+    font-size: 20px;
+  }
+
+  .sneakers {
+    margin-left: 20px;
+  }
+
+  .sneakers__description_colors {
+    flex: 1.5;
+  }
+
+  .color_mobile {
+    width: 15px;
+    height: 15px;
+  }
+
+  .history {
+    height: 380px;
+  }
+  
+  .history__title {
+    width: 260px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .history__text {
+    width: 281px;
+  }
+
+  .history__text_main {
+    font-size: 14px;
+  }
+
+  .history__video_main {
+    width: 285px;
+    height: 160px;
+  }
+}
+
+@media screen and (max-width: 374px) {
+  body {
+    max-width: 320px;
+    margin: 0 auto;
+  }
+
+  .header__container {
+    padding-left: 15px;
+    background-position-x: 30%;
+  }
+
+  .header__logo_mobile {
+    margin-left: 25px;
+  }
+
+  .menu__burger {
+    margin-right: 20px;
+  }
+  
+  .history__video {
+      padding: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5985384)